dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+    creationTimestamp: 2016-01-29T13:17:45Z
+    name: sha256:3a335d7d8a452970c5b4054ad7118ff134b3a6b50a2bb6d0c07c746e8986b28e
+    resourceVersion: "352"
+    uid: af2e7a0c-c68a-11e5-8a99-525400f25e34
+kind: ImageStreamImage
+metadata:
+  creationTimestamp: null
+  name: ruby@3a335d7
+  namespace: openshift
+  selfLink: /oapi/v1/namespaces/openshift/imagestreamimages/ruby@3a335d7
+----
+====
+
+[[image-pull-policy]]
+== Image Pull Policy
+
+Each container in a pod has a Docker image. Once you have created an image and
+pushed it to a registry, you can then refer to it in the pod.
+
+When {product-title} creates containers, it uses the container's
+`*imagePullPolicy*` to determine if the image should be pulled prior to starting
+the container. There are three possible values for `*imagePullPolicy*`:
+
+- `*Always*` - always pull the image.
+- `*IfNotPresent*` - only pull the image if it does not already exist on the node.
+- `*Never*` - never pull the image.
+
+If a container's `*imagePullPolicy*`
+parameter is not specified, {product-title} sets it based on the image's tag:
+
+. If the tag is *latest*, {product-title} defaults `*imagePullPolicy*` to `*Always*`.
+. Otherwise, {product-title} defaults `*imagePullPolicy*` to `*IfNotPresent*`.
+
+[[importing-tag-and-image-metadata]]
+== Importing Tag and Image Metadata
+
+An image stream can be configured to import tag and image metadata from an image
+repository in an external Docker image registry. To create an image stream from an external registry, set the
+`spec.dockerImageRepository` field appropriately. For example:
+
+====
+----
+apiVersion: "v1"
+kind: "ImageStream"
+metadata:
+  name: "ruby"
+spec:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+----
+====
+
+After OpenShift synchronizes the tag and image metadata, it looks something like
+this:
+
+====
+----
+kind: "ImageStream"
+apiVersion: "v1"
+metadata:
+  name: "ruby"
+  namespace: "default"
+  selfLink: "/osapi/v1/namespaces/default/imagestreams/ruby"
+  uid: "9990ea5f-f35a-11e4-937e-001c422dcd49"
+  resourceVersion: "53"
+  creationTimestamp: "2015-05-05T19:11:57Z"
+  annotations:
+    openshift.io/image.dockerRepositoryCheck: "2015-05-05T19:12:00Z"
+spec:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+status:
+  dockerImageRepository: "openshift/ruby-20-centos7"
+  tags:
+    -
+      tag: "latest"
+      items:
+        -
+          created: "2015-05-05T19:11:58Z"
+          dockerImageReference: "openshift/ruby-20-centos7:latest"
+          image: "94439378e4546d72ef221c47fe2ac30065bcc3a98c25bc51bed77ec00efabb95"
+    -
+      tag: "v0.4"
+      items:
+        -
+          created: "2015-05-05T19:11:59Z"
+          dockerImageReference: "openshift/ruby-20-centos7:v0.4"
+          image: "c7dbf059225847a7bfb4f40bc335ad7e70defc913de1a28aabea3a2072844a3f"
+----
+====
+
+Querying external registries to synchronize tag and image metadata is not
+currently an automated process. To resynchronize manually, run:
```

Comment:
That's not true anymore with `--scheduled` flag to `oc tag` or with following spec at a tag level:
```
apiVersion: v1
kind: ImageStream
metadata:
  name: ruby
spec:
  tags:
  - from: 
      kind: DockerImage
      name: openshift/ruby-20-centos7
    name: latest
    importPolicy:
      scheduled: true
```
The `importPolicy` generally allows specifying two options, `scheduled` as show above and `insecure`, which is equivalent to `--insecure` flag to `oc import-image` or `oc tag` command.